README.md

```diff
@@ -12,7 +12,6 @@ Take a look at the [Sourcegraph documentation on Batch Changes](https://docs.sou
 - [Refactor Go code with Comby](golang/comby-go-refactor/README.md)
 - [Open Jira tickets alongside changesets](ticketing-systems/jira-tickets/README.md)
 - [Open GitHub tickets alongside changesets (declarative)](ticketing-systems/github-issues/README.md)
-- [Update the Docker Hub username in Circle CI configurations](ci/update-circle-ci-config/update-circle-ci-config.batch.yaml)
```

Comment:
Ooh good catch -- we recently reorganized these and I think this one might actually be referring to the one [here](https://github.com/sourcegraph/batch-change-examples/blob/main/ci-and-tooling-config/update-circle-ci-docker-user.batch.yaml).

Apparently I can't make suggestions to deleted lines, but I think the link we're after is `ci-and-tooling-config/update-circle-ci-docker-user.batch.yaml` then.